pkg/util/env.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// EnvStr returns the value of the environment variable named by the key.
+func EnvStr(key, defaultValue string) string {
+	if value, exist := os.LookupEnv(key); exist {
```

Comment:
I suggest adding a `CD_` prefix.

Reply:
I completed it on https://github.com/cloudreve/Cloudreve/pull/1475/commits/bba4e7044a1491e7162e03dd0c356eb599d322e8